src/config.rs

```diff
@@ -989,6 +992,17 @@ impl DbConfig {
         opts.set_wal_size_limit_mb(self.wal_size_limit.as_mb());
         opts.set_max_total_wal_size(self.max_total_wal_size.0);
         opts.set_max_background_jobs(self.max_background_jobs);
+        let sanitized_background_flushes = if self.max_background_flushes == 0 {
+            1
```

Comment:
I'm thinking we should have 4 flush thread by default (one for raftdb, one for writecf, one for defaultcf, and one for the rest). But I'm not sure what's the best way to be compatible with previous config. thoughts?

Reply:
IMHO tuning up flush threads may not be ideal because L0->Ln compaction could have trouble keeping up. I think we need to be cautious here. And i don't quite understand the compatibility issue here. We can change the default value without user's knowledge.